### PR TITLE
Add sample GeoJSON data

### PR DESCRIPTION
**Pull request by jcfilben**
This PR adds some GeoJSON data for locations in Colorado.
We can add to the data or change it as needed I just wanted to get a starting point GeoJSON object out there for us to work off of.

Screen Shot 2023-06-07 at 3 00 42 PM

### CODE REVIEW

File: src/Map.jsx [hunk 1]
```diff
@@ -57,9 +58,10 @@ function Map() {
             {generateLocations(50, { center: geolocation, radius: 5 }).map(
               (location, index) => (
                 <GrommetMarker key={index} position={location} />
-              ),
+              )
             )}
           </MarkerClusterGroup>
+          <GeoJSON key="my-geojson" data={geojsonLocations} />
```
Comment (taysea) [inline src/Map.jsx:64]: quick question -- is it expected that these will be clusterable?

Comment (jcfilben) [reply src/Map.jsx:64]: Yes, we should be able to cluster the geojson point locations, haven't tested that yet though

Comment (taysea) [reply src/Map.jsx:64]: Cool -- would it be worth trying in this PR? or are you thinking separately?

Comment (jcfilben) [reply src/Map.jsx:64]: Yeah I can try it out in this PR

Comment (halocline) [reply src/Map.jsx:64]: You should be able to wrap in `<MarkerClusterGroup>` either in its own cluster group, or add it as a child of the one preceding.

Comment (jcfilben) [reply src/Map.jsx]: Just pushed up a commit that clusters the geojson markers

**Review by taysea — APPROVED** [review]

**Review by taysea — APPROVED** [review]

**Review by halocline — APPROVED** [review]

**Pull request merged**